### DISCUSSION TITLE
Chromatic adaptation matrix from arbitrary color temperature to arbitrary/standard.

### DISCUSSION THREAD

**Discussion by ssh4net** [colour-science/colour#920] [Q&A] [answered]
Hello!

Can someone point me to a right way to compute chromatic adaptation matrix from arbitrary WB color to another or standard like D50/D55/D65?

For example I have measured white point inside stage and need use that value to adapt colors to standard WB in transform data between color spaces.

And can spectral measure (I have Sekonic C-800) give me more valuable information for such tasks than estimated White Point in kelvins?

Thank you.

**Comment by thomasmansencal** [comment, marked as answer]
Hi Vlad,

The `colour.adaptation.matrix_chromatic_adaptation_VonKries` takes two *CIE XYZ* tristimulus values which are the test and reference tristimulus values to produce the adaptation matrix.

Cheers,

Thomas
